### PR TITLE
#161779741 Implement social login functionality on the client side

### PR DESCRIPTION
**Pull request by andrewinsoul**
#### What does this PR do?
This PR implements the social login functionality on the client side.

#### Description of task completed
Implement social login feature for Facebook and Google

#### How should this be manually tested?
- Run npm install
- npm start
- navigate to the login route

#### Any Background context you want to provide?
- #### N/A

#### What are the relevant Pivotal Tracker stories?
[161779741](https://www.pivotaltracker.com/story/show/161779741)

#### Screenshots(if applicable)

image

### CODE REVIEW

**Review by kingisaac95 — APPROVED** [review]

File: .eslintrc [hunk 1]
```diff
@@ -19,16 +19,19 @@
     "jest": true
   },
   "rules": {
-    "max-len": [1, 80, 2],
+    "max-len": [1, 200, 2],
```
Comment (kingisaac95) [inline .eslintrc]: Why did you change this rule? Please update it to `80`

Comment (kingisaac95) [inline .eslintrc]: Why did you change this rule? Please update it to `80`

**Review by kingisaac95 — CHANGES_REQUESTED** [review]

File: .eslintrc (hunk 1 above)
Comment (andrewinsoul) [reply .eslintrc]: I have updated it
